### DISCUSSION TITLE
Curso "Criando seu primeiro App com Flutter" > Capítulo "Todo App" > Aula "Criando o App"

### DISCUSSION THREAD

**Discussion by tonnqueiroz** [balta-io/forum#205] [Tira Dúvidas] [answered]
Boa tarde,

Estou fazendo o curso "Criando seu primeiro App com Flutter", capítulo "Todo App", aula "Criando o App" e após instalar as extensões do Flutter e do Dart, não estou conseguindo carregar um novo projeto com o comando Ctrl+Shift+P...

Conforme o print anexo, o Flutter parece estar desabilitado dentro do VSCode!

Como procedo?

PS: Todos os passos anteriores foram seguidos conforme o Curso!

![2023-01-04](https://user-images.githubusercontent.com/67775732/210633938-a59b7470-1056-4d49-9976-67a256361dda.png)

**Comment by andrebaltieri** [comment, marked as answer]
@tonnqueiroz tenta digitar flutter com F minúsculo.
Depois de instalar a extensão você reiniciou o VS Code? Tenta também reiniciar o Windows.
